### PR TITLE
docs: Fix punctuation and add image for gatsby --help 

### PR DESCRIPTION
**Pull request by sumitkharche**
## Description

1. In part zero of tutorial it is not mentioned that how to see the gatsby cli version so i added the command to find the gatsby version `gatsby --version` and a current version image.
2. Also added image for `gatsby --help` command.
3. Added the period (.) at the end of sentences whenever it is necessary.

### CODE REVIEW

File: docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -378,10 +378,10 @@ case) and you'll see your newly published site! Great work!
 
 In this section you:
 
-- Learned about Gatsby starters, and how to use them to create new projects
-- Learned about JSX syntax
-- Learned about components
-- Learned about Gatsby page components and sub-components
-- Learned about React “props” and reusing React components
+- Learned about Gatsby starters, and how to use them to create new projects.
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-one/index.md]: These are phrases and not sentences so I would avoid the periods here

(Maybe worth adding to our Style Guide at https://www.gatsbyjs.org/contributing/gatsby-style-guide @gatsbyjs/docs)

File: docs/tutorial/part-zero/index.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ _Note: Gatsby's minimum supported Node.js version is Node 8, but feel free to us
 
 1. Open your Terminal.
 1. Run `brew update` to make sure you have the latest version of Homebrew.
-1. Run this command to install Node and npm in one go: `brew install node`
+1. Run this command to install Node and npm in one go: `brew install node`.
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-zero/index.md]:
```suggestion
1. Run this command to install Node and npm in one go: `brew install node`
```
Doesn't seem necessary here

File: docs/tutorial/part-zero/index.md [hunk 3]
```diff
@@ -66,8 +66,14 @@ The Gatsby CLI tool lets you quickly create new Gatsby-powered sites and run com
 
 The Gatsby CLI is available via npm and should be installed globally by running `npm install -g gatsby-cli`.
 
+To see the latest gatsby-cli version installed run `gatsby --version`.
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-zero/index.md]:
```suggestion
To check the installed `gatsby-cli` version, run: `gatsby --version`
```

File: docs/tutorial/part-zero/index.md [hunk 4]
```diff
@@ -66,8 +66,14 @@ The Gatsby CLI tool lets you quickly create new Gatsby-powered sites and run com
...
+
+![Check gatsby version in terminal](05-gatsby-latest-version.png)
```
Comment (sidharthachatterjee) [inline docs/tutorial/part-zero/index.md]: Not sure if these are necessary in the tutorial. It's difficult to find a balance too little information and too much. 

@marcysutton What do you think?

File: docs/tutorial/part-one/index.md (hunk 1 above)
Comment (sumitkharche) [reply docs/tutorial/part-one/index.md]: Actually I thought periods make sense here. But yes your are right these are phrases not sentences .So I will remove periods.

File: docs/tutorial/part-zero/index.md (hunk 2 above)
Comment (sumitkharche) [reply docs/tutorial/part-zero/index.md]: reverted the changes

File: docs/tutorial/part-zero/index.md (hunk 3 above)
Comment (sumitkharche) [reply docs/tutorial/part-zero/index.md]: Fixed the suggestions

File: docs/tutorial/part-zero/index.md (hunk 4 above)
Comment (shannonbux) [reply docs/tutorial/part-zero/index.md]: I don't think checking the version is necessary, come to think of it. The CLI prompts users to install new versions anyway!

Comment (KyleAMathews) [reply docs/tutorial/part-zero/index.md]: Yeah agreed.

@sumitkharche could you remove all the changes except the help command? We don't add periods to the end of sentences in bullets on gatsbyjs.org so the periods were deliberately not there.

**Comment by sidharthachatterjee** [comment]
Cleaned things up and merging this in! Thank you again @sumitkharche 👍

**Pull request merged**